HATE/Info.plist

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+  <dict>
+    <key>CFBundleName</key>
+    <string>HATE</string>
+    <key>CFBundleIdentifier</key>
+    <string>com.example.HATE</string>
```

Comment:
~~I am not an example, I am the example~~ Definitely going to change this

Reply:
Haven't touched the mac stuff at all whoops.